docs/UserGuide.md

```diff
@@ -48,12 +48,13 @@ Format: `add n/[item_name] upc/[UPC] qty/[quantity] p/[price] c/[category]`
 * The `p/` parameter for `[price]` must be a **non-negative numerical** value _(decimals accepted)_.
 
 **OPTIONAL** parameters:
-* The `c/` parameter for `[category]` must be alphanumeric. (Defaults to: `Uncategorized` if not specified.)
+* The `c/` parameter for `[category]` must be alphanumeric. (Defaults to: `Uncategorized`)
 
+Example of usage: 
+=======
 
 
-
-!> **Enforced** valid range for numerical parameters is **0** to **99,999,999**.
```

Comment:
delimited with commas to make it easier for user to see, is there some reason this is changed? just curious

Reply:
Okay no worries!